libs/cdk/render-strategies/src/lib/README.md

```diff
@@ -0,0 +1,181 @@
+# RxStrategyProvider
+
+`RxStrategyProvider` is the best way to consume full power of concurrent strategies to schedule work that is not related to rendering.
```

Comment:
> to schedule work that is **not** related to rendering.

I think this sentence is not accurate since the `scheduleCD` method is about trigger rendering.

Reply:
Thank you, this is a good point. I'll think about it.

Reply:
changed to "any kind of work"